UnitsNet.Tests/CustomCode/EnergyTests.cs

```diff
@@ -78,6 +78,7 @@ public class EnergyTests : EnergyTestsBase
 
         protected override double TeraelectronVoltsInOneJoule => 6.2415093433e+6;
 
+        protected override double HorsePowerHourInOneJoule => 3.725061361111e-7;
```

Comment:
Verified 
http://www.kylesconverter.com/energy,-work,-and-heat/horsepower--hours-to-joules